crates/hyperqueue/src/common/env.rs

```diff
@@ -20,6 +20,7 @@ pub const HQ_SUBMIT_DIR: &str = create_hq_env!("SUBMIT_DIR");
 pub const HQ_ENTRY: &str = create_hq_env!("ENTRY");
 pub const HQ_PIN: &str = create_hq_env!("PIN");
 pub const HQ_TASK_DIR: &str = create_hq_env!("TASK_DIR");
+pub const HQ_ERROR: &str = create_hq_env!("ERROR");
```

Comment:
Maybe the name could be a bit more specific, like `HQ_ERROR_FILEPATH`: `ERROR_FILEPATH`?